app/views/practices/form/metrics.html.erb

```diff
@@ -0,0 +1,245 @@
+<% provide :main_classes, 'bg-gray-2' %>
+<% provide :footer_classes, 'bg-gray-2' %>
+
+<% provide :head_tags do %>
+  <%= javascript_include_tag 'metrics_page', 'data-turbolinks-track': 'reload' %>
+  <%= javascript_include_tag 'practice_editor_utilities', 'data-turbolinks-track': 'reload' %>
+  <%= javascript_include_tag 'https://www.gstatic.com/charts/loader.js' %>
+  <%= javascript_tag 'data-turbolinks-track': 'reload' do %>
+    <%= render partial: 'practices/form/metrics', formats: [:js] %>
+  <% end %>
+<% end %>
+
+<%= render 'practice_editor_sidenav' %>
+<div class="grid-container position-relative">
+  <div class="grid-row grid-gap practice-editor-min-height">
+    <div class="desktop:grid-col-3 grid-col-auto z-bottom">&nbsp;</div>
+
+    <div id="metrics" class="desktop:grid-col-9 grid-col-12 p0">
+      <div class="margin-bottom-4">
+      <%= render partial: "shared/messages", locals: {small_text: false} %>
+    </div>
+      <section class="usa-section padding-top-0 p0-bottom contact">
```

Comment:
We can get rid of the `contact` class, and we should add a margin bottom of `80px` per the designs
![Screen Shot 2021-01-13 at 10 50 39 AM](https://user-images.githubusercontent.com/34111449/104475518-330a6100-558d-11eb-8ac3-150261ef9f03.png)